server/src/main/java/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -419,7 +419,9 @@ public VpnUser doInTransaction(TransactionStatus status) {
     public boolean removeVpnUser(long vpnOwnerId, String username, Account caller) {
         final VpnUserVO user = _vpnUsersDao.findByAccountAndUsername(vpnOwnerId, username);
         if (user == null) {
-            throw new InvalidParameterValueException("Could not find vpn user " + username);
+            String errorMessage = String.format("Could not find VPN user=[%s]. VPN owner id=[%s]", username, vpnOwnerId);
+            s_logger.debug(errorMessage);
```

Comment:
```suggestion
            if(s_logger.isDebugEnabled()) {
                String errorMessage = String.format("Could not find VPN user=[%s]. VPN owner id=[%s]", username, vpnOwnerId);
                s_logger.debug(errorMessage);
            }
```

Reply:
Or we could update `log4j` to version 2. :grin:

Reply:
yes, please. to 3 by now. but as we don't use facadeproxy modules for 3rd party libraries that's not a trivial job and needs heavy sponsoring by volunteers :togueincheek: :a.k.a.: :p